src/app/contexts/UserContext/cookies/index.js

```diff
@@ -23,5 +24,6 @@ export const setPreferredVariantCookie = (service, variant) => {
   }
 
   const variantCookieName = `ckps_${getVariantCookieName(service)}`;
-  Cookie.set(variantCookieName, variant);
+  const COOKIE_EXPIRY = 365;
```

Comment:
This should be 7 days for the variant cookie
```suggestion
  const COOKIE_EXPIRY = 7;
```

Reply:
Explanation from sareh since PAL has set them to a year like with other cookies:
```
I think we want to test things to set them for a week first, just in case we messed up.
Then we can set them to a year if everything is okay.
```